governance/program/src/state/proposal.rs

```diff
@@ -465,40 +475,42 @@ impl ProposalV2 {
         account_info_iter: &mut Iter<AccountInfo>,
         realm: &Pubkey,
         realm_data: &RealmV2,
+        vote: Option<&Vote>,
```

Comment:
I'm not a huge fan of these kinds of parameters, since they can be silently ignored in some cases, ie. a veto on a community mint.  I know this isn't supported currently, but it'll help you in the long run.  I can think of a couple of options:

* Keep this, but don't make it an Option, and check that it makes sense. this will force the `None` calls into something valid, which will be weird in some situations
* Split this into `resolve_max_veto_voter_weight` and `resolve_max_governing_voter_weight`, which don't take the vote parameter.  The name of the second one kind of stinks, but it's the best I could come up with.  You could have them call this internally if you like

Reply:
Agree, I struggled with that myself. 
I fixed it by introducing `VoteKind` enum which indicates whether it's an `Electorate` vote (`Approve`, `Deny`, `Abstain`) or `Veto` vote for the opposite voting population